SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -203,6 +203,39 @@ class USpatialReceiver : public UObject
 
 	void PeriodicallyProcessIncomingRPCs();
 
+	bool NeedToLoadClass(const FString& ClassPath);
+	FString GetPackagePath(const FString& ClassPath);
```

Comment:
Static function? 

Alex makes a good point, maybe this is a useful util function that can go somewhere else.